yarn.lock

```diff
@@ -13213,7 +13206,7 @@ rc@^1.2.7, rc@^1.2.8:
     minimist "^1.2.0"
     strip-json-comments "~2.0.1"
 
-react-copy-to-clipboard@^5.0.2, react-copy-to-clipboard@^5.0.4:
+react-copy-to-clipboard@^5.0.4:
```

Comment:
Can we remove the unused `@polkadot/react-identicon` dependency as well? That way this react-copy-to-clipboard will be removed from our project.